library/build.gradle

```
@@ -4,18 +4,18 @@ apply from: "build_files/static-code-analysis.gradle"
 ext {
     PUBLISH_GROUP_ID = 'io.github.luizgrp.sectionedrecyclerviewadapter'
     PUBLISH_ARTIFACT_ID = 'sectionedrecyclerviewadapter'
-    PUBLISH_VERSION = '1.2.0'
+    PUBLISH_VERSION = '1.2.1'
```

please keep it unchanged in this pr
```suggestion
    PUBLISH_VERSION = '1.2.0'
```